tests/lib/openstack_query/runners/test_server_runner.py

```diff
@@ -1,10 +1,13 @@
 import unittest
+from openstack.exceptions import ResourceNotFound
+
 from unittest.mock import Mock, MagicMock, call, patch
 from openstack_query.runners.server_runner import ServerRunner
 
 from openstack.compute.v2.server import Server
 from openstack.identity.v3.project import Project
 from exceptions.parse_query_error import ParseQueryError
+from nose.tools import raises
```

Comment:
This circles back to shall we double down on `unitTest` for just this lib, or consider pytest?

Reply:
Ah I saw your comment above on a refresh, ACK: on Pytest when we refactor